content/_index.md

```diff
@@ -24,4 +24,189 @@ card2:
+  span: Therapy Expert 
+  facebook: index-3.html#
+  twitter: index-3.html#
+  pinterest: index-3.html#
+  email: 254@gmail.com
+teamMember3:
+  image: assets/img/team/03/team-03.jpg
+  name: Richard Sikora
+  span: CEO & Founder
+  facebook: index-3.html#
+  twitter: index-3.html#
+  pinterest: index-3.html#
+  email: sync@gmail.com
+
+counterArea1:
+  numbers: 2950
+  span: Psychologist
+counterArea2:
+  numbers: 1520
+  span: All Branches
+counterArea3:
+  numbers: 3250
+  span: Total patients
+counterArea4:
+  numbers: 2150
+  span: Achievements
+
+whyChooseArea:
+  span: Why choose us
+  title: Best Method Apply our Patients
+  list1: Psychodynamic Therapy
+  list2: Personal Development
+  list3: Motivational Speech
+  list4: Mindtherafy Professional
+  list5: Assesment Consultation
+  image: assets/img/why/why-img-03.jpg
+
+imageTestimonialArea:
+  image: assets/img/testimonial/testimonial-img.jpg
+  imageHeadline: See Our Success Story
+
+testimonialFeedback:
+  span: patients feedbak
+  title: Happy Patient Feedback About our Service
+
+patientFeedback1:
+  description: Malesuada imperdiet commodo nostra taciti neque arcu in sem a
+                  vivamus tempor ac sem convallis ullamcorper ac enim Massa
+                  massa porta interdum to nullam nis augue donec leo ut
+                  fermentum senectus eget. Metus lacinia turpis. Potenti in
+                  aliquet ipsum primis a pede volutpat. Feugiat hymenaeos
+                  facilisis erat morbi libero to sociosqu inte dum justo
+                  sagittis feugiat facilisis morb.
+  image: assets/img/testimonial/person-1.png
+  customerName: Eleonora Morgan
+  locationName: CEO & Founder
+
+patientFeedback2:
+  description: Malesuada imperdiet commodo nostra taciti neque arcu in sem a
+                  vivamus tempor ac sem convallis ullamcorper ac enim Massa
+                  massa porta interdum to nullam nis augue donec leo ut
+                  fermentum senectus eget. Metus lacinia turpis. Potenti in
+                  aliquet ipsum primis a pede volutpat. Feugiat hymenaeos
+                  facilisis erat morbi libero to sociosqu inte dum justo
+                  sagittis feugiat facilisis morb.
+  image: assets/img/testimonial/person-2.png
+  customerName: Jeso Smith
+  locationName: CEO & Founder
+
+patientFeedback3:
+  description: Malesuada imperdiet commodo nostra taciti neque arcu in sem a
+                  vivamus tempor ac sem convallis ullamcorper ac enim Massa
+                  massa porta interdum to nullam nis augue donec leo ut
+                  fermentum senectus eget. Metus lacinia turpis. Potenti in
+                  aliquet ipsum primis a pede volutpat. Feugiat hymenaeos
+                  facilisis erat morbi libero to sociosqu inte dum justo
+                  sagittis feugiat facilisis morb.
+  image: assets/img/testimonial/person-3.png
+  customerName: Salim rana
+  locationName: CEO & Founder
+
+contactArea:
```

Comment:
remove contact this and make this static

Reply:
done